Mathlib/Analysis/Meromorphic/NormalForm.lean

```diff
@@ -284,12 +288,15 @@ lemma MeromorphicAt.eqOn_compl_singleton_toMermomorphicNFAt (hf : MeromorphicAt
   simp [toMeromorphicNFAt, hf]
 
 /-- Conversion to normal form at `x` changes the value only at x. -/
-lemma MeromorphicAt.eq_nhdNE_toMeromorphicNFAt (hf : MeromorphicAt f x) :
+lemma MeromorphicAt.eq_nhdsNE_toMeromorphicNFAt (hf : MeromorphicAt f x) :
     f =ᶠ[𝓝[≠] x] toMeromorphicNFAt f x :=
   eventually_nhdsWithin_of_forall (fun _ hz ↦ hf.eqOn_compl_singleton_toMermomorphicNFAt hz)
 
+@[deprecated (since := "2025-05-22")]
+alias MeromorphicAt.eq_nhdNE_toMeromorphicNFAt := MeromorphicAt.eq_nhdsNE_toMeromorphicNFAt
+
 /-- Two analytic functions agree on a punctured neighborhood iff they agree on a neighborhood. -/
-private lemma AnalyticAt.eventuallyEq_nhdNE_iff_eventuallyEq_nhd {g : 𝕜 → E} {z₀ : 𝕜}
+lemma AnalyticAt.eventuallyEq_nhds_of_nhdsNE {g : 𝕜 → E} {z₀ : 𝕜}
```

Comment:
I would leave this lemma as `private`, because the non-`private` version should be stated for `ContinuousAt` functions, not `AnalyticAt`.
Otherwise LGTM. Thanks!
bors d+

Reply:
Then I've simply removed the lemma, because it's not used anywhere and a private lemma which is not used is not very useful...